packages/jest-core/package.json

```diff
@@ -41,7 +41,7 @@
     "@types/exit": "^0.1.30",
     "@types/graceful-fs": "^4.1.2",
     "@types/micromatch": "^3.1.0",
-    "@types/p-each-series": "^1.0.0",
+    "@types/p-each-series": "1.0.0",
```

Comment:
newer versions of `p-each-series` has types, but they are node 8 only. for some reason the types published as `1.0.2` makes type check fail for us...